### PR TITLE
Archive patest

### PR DESCRIPTION
**Pull request by Ahzyuan**
Archive package test

### CODE REVIEW

File: check_meta.sh
Comment (SquirrelStore) [inline check_meta.sh:1]: What's the function of this file?

**Review by Ahzyuan — COMMENTED** [review]
don't hesitate to ask anything if you have a question

**Review by SquirrelStore — CHANGES_REQUESTED** [review]
is python 3.6 allowed?

File: check_meta.sh
Comment (Ahzyuan) [reply check_meta.sh:1]: To check whether the metadata is valid and the same as expected

**Comment by Ahzyuan** [comment]
yes, 3.6 is allowed, so I don't think should be modified.

File: packaging.sh
Comment (SquirrelStore) [inline packaging.sh:1]: what is this

Comment (Ahzyuan) [reply packaging.sh:1]: a bash script to build distribution(`.tar.gz` + `.whl`)

File: patest/__init__.py
Comment (SquirrelStore) [inline patest/__init__.py:1]: Accept

File: MANIFEST.in
Comment (SquirrelStore) [inline MANIFEST.in:1]: Accept

File: check_meta.sh
Comment (SquirrelStore) [inline check_meta.sh:1]: Accept

File: packaging.sh
Comment (SquirrelStore) [inline packaging.sh:1]: Accept

File: patest/__main__.py
Comment (SquirrelStore) [inline patest/__main__.py:1]: Accept

File: patest/cli.py
Comment (SquirrelStore) [inline patest/cli.py:1]: Accept

File: patest/core.py
Comment (SquirrelStore) [inline patest/core.py:1]: Accept

File: patest/greet.py
Comment (SquirrelStore) [inline patest/greet.py:1]: Accept

File: requirements.txt
Comment (SquirrelStore) [inline requirements.txt:1]: Accept

File: ruff.toml
Comment (SquirrelStore) [inline ruff.toml:1]: Accept

File: setup.cfg
Comment (SquirrelStore) [inline setup.cfg:1]: Accept

File: setup.py
Comment (SquirrelStore) [inline setup.py:1]: Accept

**Review by SquirrelStore — COMMENTED** [review]
Ok, we will merge as soon as possible

**Review by SquirrelStore — APPROVED** [review]

**Pull request merged**